### ISSUE TITLE
Bug: re initialise the array object in JSON

### ISSUE THREAD

**Issue by mehmetdhhs** [domaindrivendev/Swashbuckle#1237]
Hi,

I am creating a json array object manually as I am building up a custom model on Swagger, such as;
1 - 
`swaggerDoc.properties["JsonData"].properties.Add("Parent", schemaArray);`

Once the parent object array added, then I define the items are going to be set right after the following snippet code:
2 - 
`swaggerDoc.properties["JsonData"].properties["Parent"].items = new Schema() { properties = new Dictionary<string, Schema>() };`

Then I do insert the child items underneath the above Parent object array:
3 - 
`swaggerDoc.properties["JsonData"].properties["Parent"].items.properties.Add("Child1", schemaChild);`

Let say I do have another Parent2 object array and I do need to repeat the above flow, however, once I register the second Parent2 object array within the JSON, and re-initialise the items object that removes the existing items for Parent object array that was already created and assigned.

Basically, every object array contains the same items.properties values

There must be a way of binding different items under each / different object array, is there a way of doing that? thanks
